src/test/resources/retest/recheck/de.retest.web.it.SimpleRecheckShowcaseIT/simpleShowcase.index.recheck/retest.xml

```diff
@@ -11350,10 +11408,6 @@
 					</attributes>
 				</containedElements>
 			</containedElements>
-			<screenshot>
-				<persistenceId>html</persistenceId>
```

Comment:
Also this: https://github.com/retest/recheck-web/blob/master/src/test/resources/retest/recheck/de.retest.web.it.SimpleRecheckShowcaseIT/simpleShowcase.index.recheck/screenshot/html.png

Reply:
Removed in the current iteration.